platform/android/MapLibreAndroidTestApp/src/androidTest/java/org/maplibre/android/offline/OfflineDownloadTest.kt

```diff
@@ -16,6 +17,7 @@ import java.util.concurrent.TimeoutException
 /**
  * Integration test that validates downloading an offline region from a point geometry at zoomlevel 17
  */
+@Ignore("https://github.com/maplibre/maplibre-native/issues/2318")
```

Comment:
Thanks for creating these issues so we don't forget to come back and look at these flaky tests :)